x-pack/plugins/observability_solution/asset_manager/server/templates/assets_template.ts

```diff
@@ -6,11 +6,13 @@
  */
 
 import { IndicesPutIndexTemplateRequest } from '@elastic/elasticsearch/lib/api/types';
+import { ASSETS_INDEX_PREFIX } from '../constants';
 
 export const assetsIndexTemplateConfig: IndicesPutIndexTemplateRequest = {
   name: 'assets',
   priority: 100,
   data_stream: {},
+  index_patterns: [`${ASSETS_INDEX_PREFIX}*`],
```

Comment:
this template is unused by the transform based outputs, right? removing this would be part of the clean up of the old 'asset' stuff?

Reply:
Yeah... we need to remove that.